packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js

```diff
@@ -114,14 +114,13 @@ class S78Journey extends Journey {
 				.addQuestion(questions.representationUpload)
 				.withCondition(questionHasAnswer(questions.representationsFromOthers, 'yes')),
 			new Section('Planning officer’s report and supporting documents', 'planning-officer-report')
+				.addQuestion(questions.uploadDevelopmentPlanPolicies)
```

Comment:
reordered this as per the request in this [navigation correction document](https://pins-ds.atlassian.net/jira/software/c/projects/AAPD/boards/213?selectedIssue=AAPD-940)